index.html

```diff
@@ -118,34 +118,44 @@ <h4>OGC Implementation</h4>
+      <p class="text-left">
+        GeoServer participates in the annual <a href="https://foss4g.org/">Free and Open Source Software for Geospatial</a> conference, Join us for <a href="https://osgeo.org/projects/geoserver/">FOSS4G 2022 - Firenze</a>!
+      </p>
+      <a href="https://osgeo.org/projects/geoserver/">
+        <img src="img/OSGeo_project.png" width="150" height="58"/>
+      </a>
+      <a href="https://2022.foss4g.org/">
+        <img src="img/foss4g.png" width="150" height="54">
+      </a>
     </div>
   </div>
 </section>
 
 <section id="thanks" class="text-center">
   <h2>Special Thanks</h2>
-  <p class="lead">The following products are kindly supporting open source projects like GeoServer.</p>
+  <p class="lead"></p>
   <div class="row">
-    <div class="col-md-6" id="books">
-       <a href="http://www.packtpub.com/geoserver-share-edit-geospatial-data-beginners-guide/book"><img src="img/books.png"/ ></img></a>
+    <div class="col-md-6" id="sponsor">
        <p class="text-left">
-         <a href="https://www.packtpub.com/application-development/geoserver-beginner’s-guide">GeoServer Beginner's Guide</a>, <a href="https://www.packtpub.com/application-development/geoserver-cookbook">GeoServer Cookbook</a>, <a href="https://www.packtpub.com/networking-and-servers/mastering-geoserver">Mastering GeoServer</a>, <a href="https://www.packtpub.com/application-development/geoserver-beginner%E2%80%99s-guide-second-edition">GeoServer Beginner's Guide - Second Edition</a> <a href="https://www.packtpub.com/application-development/fundamentals-geoserver-video">Fundamentals of Geoserver [Video]</a> and <a href="https://www.packtpub.com/application-development/expert-geoserver">Expert Geoserver</a> are available from Packt Publishing.
-       </p>
-     </div>
-     <div class="col-md-6">
-       <a href="https://www.yourkit.com/java/profiler/index.jsp"><img src="img/yklogo.png"/ ></img></a>
+          Thanks for your <a href="https://github.com/geoserver/geoserver/wiki/Sponsor">sponsorship</a> and <a href="https://github.com/sponsors/OSGeo">github</a> / <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4XZ76BZMEWZ24&source=url">paypal</a> donations. Financial support contributes to project sustainability by supporting underfunded <a href="https://github.com/geoserver/geoserver/wiki/Maintenance">maintenance</a>, <a href="https://github.com/geoserver/geoserver/blob/main/SECURITY.md">security</a>, and <a href="https://github.com/geoserver/geoserver/wiki/Restore-CITE-Certification">interoperability</a> activities.
+      </p>
+      <a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=4XZ76BZMEWZ24&source=url">
+        <img src="img/btn_donateCC_LG.gif" width="147" height="47"/>
+      </a>
+    </div>
+    <div class="col-md-6" id="in-kind">
+      <p class="text-left">
+         GeoServer benefits the following development tools via their in-kind support of  open source projects.
```

Comment:
The sentence doesn't really match up - we don't benefit the tools, we benefit from them, but the tools aren't providing support - the companies are.
Maybe
```suggestion
         GeoServer is one of many open-source projects that receives in-kind support from:
```